packages/components/src/Checkbox/Checkbox.test.tsx

```diff
@@ -19,6 +20,23 @@ it("renders a disabled Checkbox", () => {
   expect(tree).toMatchSnapshot();
 });
 
+it("renders a checkbox with an empty element", () => {
+  const tree = renderer
+    .create(
+      <Checkbox>
+        <></>
+      </Checkbox>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it("renders a checkbox with children", () => {
+  const childContent = <Text>Content</Text>;
+  const tree = renderer.create(<Checkbox>{childContent}</Checkbox>).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
we generally avoid using snapshots in atlantis because of how it's hard to know exactly what you're testing. (Sorry if this is not clear, we're working on clearing that up soon)

This should make it more specific on what exactly are you testing.

```suggestion
it("renders a checkbox with children", () => {
  const { getByText } = render(<Checkbox>{<Text>Content</Text>}</Checkbox>);
  expect(getByText("Content")).toBeInTheDocument();
});
```

Reply:
Sounds good! I have also changed some of the other tests to not use snapshots anymore as well

Reply:
@ChristianLee-Jobber thank you sir! Muchas gracias!